libcnb-proc-macros/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "libcnb-proc-macros"
```

Comment:
We talked about "libcnb" not being a great name for the higher level concepts recently. If we wanted to switch to another naming strategy, it seems that doing so before we have too many more crates published might be best. Though if we were going to keep libcnb for the lower-level/data structure components, then I presume `libcnb-proc-macros` might still be a our preferred name even after a naming shuffle, so therefore doesn't affect this PR?

Reply:
Yeah, I think it's fine for this PR.